src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPayment = 40;
+  const discountBasic = 20;
+  const discountBig = 50;
+
+  if (days < 7 && days > 2) {
+    return (days * dailyPayment) - discountBasic;
```

Comment:
```suggestion
    return rentalPrice - discountBasic;
```
Would be great to create a variable for that value in order not to duplicate yourself and make your code cleaner